packages/python-names/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname=python-names
+pkgver=0.3.0
+pkgrel=1
+pkgdesc='Random name generator.'
+arch=('any')
+url='https://pypi.org/project/names'
```

Comment:
Should use the github url.

https://github.com/treyhunner/names

Reply:
Complete.